tests/snapshots/add/add_basic/mod.rs

```diff
@@ -0,0 +1,24 @@
+use cargo_test_support::compare::assert;
+use cargo_test_support::prelude::*;
+use cargo_test_support::Project;
+use crate::cargo_add::init_registry;
+
+#[cargo_test]
+pub fn add_basic() {
+    init_registry();
+    let project = Project::from_template("tests/snapshots/add/add_basic/in");
+    let project_root = project.root();
+    let cwd = &project_root;
+
+
+    snapbox::cmd::Command::cargo()
```

Comment:
Nit: how come an extra newline got inserted?

Reply:
Not sure, ill remove it